### PR TITLE
Created integral pizza menu

### LINKED ISSUE #4: Improve integrity of the data
And add prices

### PR DESCRIPTION
**Pull request by Paulina100**

### CODE REVIEW

File: data/Pizzas.csv [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
-28;Kebab;sos pomidorowy, ser, oregano, kurczak, cebula, ogórek kiszony, ogórek świeży, pomidor, pekińska
-29;Przysmak Biskupa;sos pomidorowy, ser, oregano, salami, jalapeno, świeży czosnek i cebula
-30;Primavera;sos pomidorowy, ser, oregano, sambal (sos ostry), salami poprószone pieczarkami
-31;BBQ z kurczakiem;sos BBQ, boczek, grillowany kurczak, czerwona cebula, czosnek, papryka, kukurydza
-32;BBQ z wołowiną;sos BBQ, boczek, wołowina, czerwona cebula, czosnek, jalapeno, kukurydza
-33;Carbonara;sos czosnkowy, boczek, cebula, czosnek, jajko, parmezan, świeża pietruszka
-34;Broccoli;sos czosnkowy, brokuły, kukurydza, ser feta, mięso kebab
-35;Farmerska;sos pomidorowy, ser, oregano, boczek, salami, kiełbasa, cebula, kukurydza, papryka
-36;Verdura;ser feta, szpinak, oliwki, czerwona cebula
+﻿pizza_id;pizza_name;ingredients;price
```
Comment (Paulina100) [inline data/Pizzas.csv]: If you add ; at the begining it should work, idk not tested

Comment (Paulina100) [reply data/Pizzas.csv]: Also not sure about , in numbers

**Comment by pietruszkowiec** [comment]
Change comma float number notation to dot notation, i.e. 35,6 -> 35.6
Currently the data breaks the following line:
Screen Shot 2023-03-21 at 22 53 42

**Review by pczarnik — CHANGES_REQUESTED** [review]
Change comma float number notation to dot notation, i.e. 35,6 -> 35.6
Currently the data breaks the following line:
Screen Shot 2023-03-21 at 22 55 56

**Review by pczarnik — APPROVED** [review]

**Pull request merged**